fe/fe-core/src/main/java/org/apache/doris/mysql/privilege/Auth.java

```diff
@@ -1937,4 +1938,97 @@ public String getDefaultCloudCluster(String user) {
         return cluster;
     }
     // ====== END CLOUD ======
+
+    // for mysql.user table
+    public List<List<String>> getAllUserInfo() {
```

Comment:
UserInfo is hard code data, why not fill it in be?

Reply:
Because the data in userinfo requires permissions and password policies, FE can better directly determine the data that needs to be sent.